tests/golden/test_mapper_allNN_last.qasm

```diff
@@ -1,82 +1,59 @@
-# Generated by OpenQL 0.10.0 for program test_mapper_allNN
+# Generated by OpenQL 0.11.1 for program test_mapper_allNN
 version 1.2
 
 pragma @ql.name("test_mapper_allNN")
 
 
 .kernel_allNN
-    y90 q[3]
-    { # start at cycle 1
+    { # start at cycle 0
         x q[0]
         x q[3]
     }
-    cz q[0], q[3]
+    cnot q[0], q[3]
+    skip 2
     x q[1]
-    { # start at cycle 4
-        cz q[1], q[3]
-        y90 q[2]
-    }
-    x q[2]
-    cz q[0], q[2]
-    y90 q[5]
+    cnot q[1], q[3]
+    skip 2
     { # start at cycle 8
-        y90 q[3]
+        x q[2]
         x q[5]
     }
-    cz q[3], q[5]
-    ym90 q[0]
-    { # start at cycle 11
-        y90 q[4]
-        cz q[3], q[0]
+    { # start at cycle 9
+        cnot q[0], q[2]
+        cnot q[3], q[5]
     }
-    { # start at cycle 12
+    skip 3
+    cnot q[3], q[0]
+    skip 2
+    { # start at cycle 16
         x q[4]
-        y90 q[6]
-    }
-    { # start at cycle 13
-        cz q[1], q[4]
         x q[6]
     }
-    cz q[3], q[6]
-    { # start at cycle 15
-        ym90 q[1]
-        y90 q[2]
-        y90 q[4]
+    { # start at cycle 17
+        cnot q[1], q[4]
+        cnot q[3], q[6]
     }
-    { # start at cycle 16
-        cz q[3], q[1]
-        cz q[2], q[5]
-    }
-    cz q[4], q[6]
-    { # start at cycle 18
-        y90 q[5]
-        ym90 q[3]
-    }
-    cz q[5], q[3]
-    skip 1
+    skip 3
     { # start at cycle 21
-        cz q[2], q[0]
-        cz q[4], q[1]
-        y90 q[6]
-    }
-    cz q[6], q[3]
-    { # start at cycle 23
-        ym90 q[2]
-        ym90 q[4]
+        cnot q[3], q[1]
+        cnot q[2], q[5]
+        cnot q[4], q[6]
     }
-    { # start at cycle 24
-        cz q[5], q[2]
-        cz q[6], q[4]
+    skip 3
+    cnot q[5], q[3]
+    skip 3
+    { # start at cycle 29
+        cnot q[2], q[0]
+        cnot q[4], q[1]
+        cnot q[6], q[3]
     }
-    skip 1
-    { # start at cycle 26
-        y90 q[0]
-        y90 q[1]
-        y90 q[2]
-        y90 q[3]
-        y90 q[4]
+    skip 3
+    { # start at cycle 33
+        cnot q[5], q[2]
+        cnot q[6], q[4]
     }
-    { # start at cycle 27
+    skip 3
```

Comment:
As written elsewhere, difference also includes not decomposing cnot. This makes resource use overly conservative and thus the circuit more sequential.
That can explain the 10 cycles increase.